src/domain/venue/venueInfo/VenueInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import TextTitle from '../../../common/components/textTitle/TextTitle';
+import { useVenueQuery } from '../../../generated/graphql';
+import IconFood from '../../../icons/IconFood';
+import IconGarderobe from '../../../icons/IconGarderobe';
+import { Language } from '../../../types';
+import { getVenueDescription } from '../utils';
+import styles from './venueInfo.module.scss';
+
+interface Props {
+  language: Language;
+  placeId: string;
+}
+
+const VenueInfo: React.FC<Props> = ({ language, placeId }) => {
+  const { t } = useTranslation();
+  const { data: venueData } = useVenueQuery({ variables: { id: placeId } });
+  const venueDescription = getVenueDescription(venueData, language);
+  const hasClothingStorage = venueData?.venue?.hasClothingStorage;
+  const hasSnackEatingPlace = venueData?.venue?.hasSnackEatingPlace;
```

Comment:
Maybe we could have getVenueFields (similar as getEventFields) helper function to get all these values.